src/main/java/shrek/Shrek.java

```diff
@@ -0,0 +1,476 @@
+
+
+public class Shrek {
+    private static final ArrayList<UserContent> lists = new ArrayList<>();
+    private static final String LINE = PrintStrings.LINE;
+    private static final int INDEX_OF_TASK_CONTENT = 0;
+    private static final int INDEX_OF_TASK_NAME = 1;
+    private static final int INDEX_OF_TASK_COMMAND = 0;
+    private static final int INDEX_OF_TASK_INPUT = 1;
+    private static final int FIND_TASK_OR_TIME = 0;
+    private static final int FIND_CONTENT = 1;
+    private static final int INDEX_OF_TASK_MARKED = 0;
+    private static final int INDEX_OF_TIME_INPUT = 1;
+    private static final int INDEX_OF_DATE_INPUT = 0;
+    private static final int NUMBER_OF_TERMS_IN_SPLIT = 2;
+    private static final int LIST_INDEX_CORRECTION = -1;
+    public static final String NEW_LINE = System.lineSeparator();
+    private static int errorCount = 0;
+    private static final String[] listOfCommands = {"todo", "deadline", "event", "mark", "unmark", "delete", "save", "find", "list"};
+    private static String OUTPUT_FILE_PATH;
+    public static final String ANSI_GREEN = "\u001B[32m";
+    public static final String ANSI_RESET = "\u001B[0m";
+
+    public static void initialiseShrek() throws InvalidCommandException {
+        try {
+            String currentDirectory = System.getProperty("user.dir");
+            OUTPUT_FILE_PATH = currentDirectory + "/data/output.txt";
+            File dataDirectory = new File(currentDirectory + "/data");
+            if (!dataDirectory.exists()) {
+                dataDirectory.mkdir();
+            }
+            File outputFile = new File(OUTPUT_FILE_PATH);
+            if (!outputFile.createNewFile()) {
+                loadFromOutput();
+            }
+        } catch (IOException e) {
+            throw new InvalidCommandException("IO excepts you!", errorCount);
+        }
+    }
+
+    public static String convertMark(UserContent task) {
+        String mark;
+        if (task.getMark()) {
+            mark = "marked";
+        } else {
+            mark = "unmarked";
+        }
+        return mark;
+    }
+
+    public static void loadFromOutput() throws FileNotFoundException {
+        File outputFile = new File(OUTPUT_FILE_PATH);
+        Scanner lineScanner = new Scanner(outputFile);
+        while (lineScanner.hasNext()) {
+            String[] splitMarkAndContents = lineScanner.nextLine().split(" ", NUMBER_OF_TERMS_IN_SPLIT);
+            takeInput(splitMarkAndContents[INDEX_OF_TASK_INPUT], false);
+            if (splitMarkAndContents[INDEX_OF_TASK_MARKED].equals("marked")) {
+                lists.get(lists.size() + LIST_INDEX_CORRECTION).setMark();
+            }
+        }
+    }
+
+    public static void saveToOutput() throws InvalidCommandException {
+        try {
+            clearOutput();
+            for (UserContent task : lists) {
+                String taskName = task.getTaskName();
+                switch (taskName) {
+                case "T":
```

Comment:
Avoid using magic literals such as `"T"`, consider extracting it out into a constant.